cmd/bootstrapper/uninstall_kubelet.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"flag"
+	"os"
+
+	"github.com/spf13/cobra"
+
+	"github.com/openshift/windows-machine-config-bootstrapper/pkg/bootstrapper"
+)
+
+var (
+	// uninstallKubeletCmd describes the uninstall-kubelet command
+	uninstallKubeletCmd = &cobra.Command{
+		Use:   "uninstall-kubelet",
+		Short: "Uninstalls the kubelet service from the Windows node",
+		Long:  "Uninstalls the kubelet service from the Windows node",
+		Run:   runUninstallKubeletCmd,
+	}
+)
+
+func init() {
+	rootCmd.AddCommand(uninstallKubeletCmd)
+}
+
+// runUninstallKubeletCmd uninstalls kubelet service from the Windows node
+func runUninstallKubeletCmd(cmd *cobra.Command, args []string) {
+	flag.Parse()
+	wmcb, err := bootstrapper.NewWinNodeBootstrapper("", "", "", "", "")
+	if err != nil {
+		log.Error(err, "could not create bootstrapper")
+		os.Exit(1)
+	}
+
+	// uninstall kubelet Windows service
+	err = wmcb.UninstallKubelet()
+	if err != nil {
+		log.Error(err, "could not uninstall kubelet service")
+		os.Exit(1)
+	}
+
+	// Send success message to StdOut to ascertain that kubelet removal was successful
+	os.Stdout.WriteString("kubelet service uninstalled successfully")
+
+	err = wmcb.Disconnect()
+	if err != nil {
```

Comment:
nit: I would prefer using this format if  nothing but err is being returned
```suggestion
	if err = wmcb.Disconnect(); err != nil {
```

Reply:
Applies to https://github.com/openshift/windows-machine-config-bootstrapper/pull/259/files#diff-7bd684eea45943c86e1fa622be0ac5b58ebb684b7059e40d5b332b942b5ed64aR36 as well
`err = wmcb.UninstallKubelet()`